### PR TITLE
Add components to discover page

### PR DESCRIPTION
**Pull request by Johannafendrich**
> Does not yet provide the correct functionality. This is what I will do next.
Preview:  https://my-homie-discoverpage-6spyjrhx.herokuapp.com/home
#57

### CODE REVIEW

File: client/src/components/MenueCard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+const Img = styled.img`
+  width: 300px;
+  height: auto;
+  border-radius: 1.5rem;
+  object-fit: cover;
+  position: relative;
+`;
+
+const MenueCard = ({ src, alt }) => (
```
Comment (marwinburesch) [inline client/src/components/MenueCard.js:34]:
```suggestion
const MenuCard = ({ src, alt }) => (
```

File: client/src/components/MenueCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const InputContainer = styled.div`
```
Comment (marwinburesch) [inline client/src/components/MenueCard.js:5]: why is it called an `InputContainer` I don't see any `<input>` elements 🧐

File: client/src/components/MenueCard.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  min-width: 300px;
+  height: auto;
+  margin-top: 0.4rem;
+  border-radius: 1.5rem;
+  color: #323338;
+  box-shadow: 0px 10px 15px rgba(0, 0, 0, 16%);
+`;
+
+const Img = styled.img`
+  width: 300px;
```
Comment (marwinburesch) [inline client/src/components/MenueCard.js:27]: I recommend moving `width` to the parent element :)

File: client/src/components/MenueCard.js (hunk 2 above)
Comment (Johannafendrich) [reply client/src/components/MenueCard.js:5]: Your right it isn't very obvious. I changed that :)

File: client/src/components/MenueCard.js (hunk 3 above)
Comment (Johannafendrich) [reply client/src/components/MenueCard.js:27]: I don't know why but the picture will be huge, so that's why I gave the width to the picture itself 😅

**Pull request merged**